src/main/java/com/aws/greengrass/mqtt/bridge/auth/MQTTClientKeyStore.java

```diff
@@ -132,6 +136,23 @@ public void updateCA(List<String> caCerts) throws IOException, CertificateExcept
         updateListeners.forEach(UpdateListener::onCAUpdate); //notify MQTTClient
     }
 
+    /**
+     * Get the root CA cert from the key store.
+     *
+     * @return root ca cert
+     * @throws KeyStoreException if keystore has not been initialized
+     * @throws CertificateEncodingException if an encoding error occurs
+     */
+    public Optional<String> getRootCACert() throws KeyStoreException, CertificateEncodingException {
+        Certificate certificate = keyStore.getCertificate("CA0");
```

Comment:
what if there's a chain? Don't we need the whole cert chain?

Reply:
We already have the list of certs as pem encoded being passed to `updateCA`, can we just use that instead of trying to extract them from this keystore?

Reply:
ah i thought it validated for chain length of 1...not the case, will adjust

Reply:
What is doing this validation?

Reply:
in crt TLSContextOptions, `PemUtils.sanityCheck`.  when setting the client cert & key (via TlsContextOptions.createWithMtlsJavaKeystore), it checks for chain length of 1 (TlsContextOptions#initMtls), so i wrongly assumed the same for TlsContextOptions#overrideDefaultTrustStore

Reply:
updated to use full chain